docs/robots/husky/user_manual_husky.mdx

```diff
@@ -127,280 +164,62 @@ The connectors used in the Husky's user bay are Weidmüller's 6 Position Termina
 | Communication                  | RS-232, 115200 baud                                                      |
 | Wheel Encoders                 | 78,000 ticks/m                                                           |
 | Internal Sensing               | Battery Status, Wheel Odometry, Motor Currents                           |
+| Environmental                  | IP 44 (upgrade to IP 55 available)                                       |
```

Comment:
IP44 could be misleading. I would not use an uncovered Husky outside when it is raining. The User Bay would be filled with water.

Consider:

- **IP30** for base husky with a computer
- **IP55** for a Husky with the weatherproofing kit

Reply:
Updated